src/index.md

```diff
@@ -9,7 +9,7 @@ styles:
 
 <div class="content-sizing" markdown="1">
 
-# Installation
+# Getting Started
```

Comment:
Oh, sorry. 'Get Started'. I had this in another PR https://github.com/GoogleChrome/workbox-microsite/pull/15